regexes/device/mobiles.yml

```diff
@@ -357,30 +357,47 @@ Apple:
 
 # micromax
 MicroMax:
-  regex: 'MicroMax[ \-\_]?[a-z0-9]+'
+  regex: '(MicroMax[ \-\_]?[a-z0-9]+|Q327)'
   device: 'smartphone'
   models:
     - regex: 'MicroMax(?:[ \-\_])?(P[a-z0-9]+)'
       model: '$1'
       device: 'tablet'
     - regex: 'MicroMax(?:[ \-\_])?([a-z0-9]+)'
       model: '$1'
+    - regex: 'Q327'
+      model: 'Bolt'
 
 # 3Q
 3Q:
   regex: '(AC0731B|AC1024C|AC7803C|BC9710AM|EL72B|LC0720C|LC0723B|LC0725B|LC0804B|LC0808B|LC0809B|LC0810C|LC0816C|LC0901D|LC1016C|MT0724B|MT0729B|MT0729D|MT0811B|OC1020A|RC0709B|RC0710B|RC0718C|RC0719H|RC0721B|RC0722C|RC0726B|RC0734H|RC0743H|RC0817C|RC1018C|RC1019G|RC1025F|RC1301C|RC7802F|RC9711B|RC9712C|RC9716B|RC9717B|RC9724C|RC9726C|RC9727F|RC9730C|RC9731C|TS0807B|TS1013B|VM0711A|VM1017A|RC0813C|QS9719D|QS9718C|QS9715F|QS1023H|QS0815C|QS0730C|QS0728C|QS0717D|QS0716D|QS0715C|MT7801C)'
   device: 'tablet'
   model: '$1'
 
+# 4Good
+4Good:
+  regex: '(?:4Good)?(S450m [43]G|S555m 4G|S501m 3G|T700i_3G|Light A103)'
+  device: 'smartphone'
```

Comment:
If I see that right all devices from 4Good start with `4Good`, so maybe we could make that detection more generic like:
```
regex: '4Good|S450m [43]G|S555m 4G|S501m 3G|T700i_3G|Light A103'
```

And add a generic model detection below:
```
- regex: '4Good[ _]([^;/]+) Build'
  model: '$1'
- regex: '4Good[ _]([^);/ ]+)'
  model: '$1'
```